hashicorp/boundary-ui
# feat: 🎸 Add initial setup for client daemon integration

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-11702

## Description
Initial setup for the client daemon integration. On startup, we will start the daemon and when the app quits, we'll shut it down. When a user initially logs in OR if the user's session is restored, we will add the token to the daemon. If it's the same token, the daemon will just update it.

## How to Test
Build a binary from the `llb-client-daemon` branch and use that binary in your local dev environment. You can confirm in your terminal that a `boundary daemon status` returns a user added to the daemon after logging in.

## Checklist:
